labgrid/driver/usbstoragedriver.py

```diff
@@ -111,7 +111,7 @@ def copy_files(self, filenames, target_dir, partition=1):
                 mf = ManagedFile(f, self.storage)
                 mf.sync_to_resource()
                 self.logger.debug("Copy %s to %s", mf.get_remote_path(), target_path)
-                cp_args = ["cp", "-t", target_path, mf.get_remote_path()]
+                cp_args = ["cp", "-t", str(target_path), mf.get_remote_path()]
```

Comment:
Oh, apparently ``subprocess.Popen`` accepts path-like objects only starting from Python 3.6.

Reply:
Yes, the woes of python 3.5 support.